web/templates/statistics/index.html.eex

```diff
@@ -1,6 +1,24 @@
-<h1>Statistics for /<%= @statistics.slug %></h1>
+<h1>Statistics for /<%= @mapped_url.slug %></h1>
+
+<%= link(to: mapped_url_path(@conn, :index)) do %>
+  Back
+<% end %>
 
 <dl data-role="statistics">
-  <dt>Views</dt>
-  <dd><%= @statistics.views %></dd>
+  <table>
+    <thead>
+      <tr>
+        <th>Month</th>
+        <th>Requests</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for statistics <- @mapped_url.statistics do %>
+        <tr>
+          <td><%= statistics.month |> format %></td>
```

Comment:
![](https://s-media-cache-ak0.pinimg.com/736x/99/95/fc/9995fc5ecf7abe34582a61c8205a295e.jpg)


When should you curry and when should you invoke directly?